src/client/routes/nfc/nfc.graphql.ts

```diff
@@ -1,14 +1,6 @@
 import gql from 'graphql-tag';
 
 export default gql`
-	query events {
```

Comment:
???

Reply:
Redundant query, I moved this query to events/events.graphql.ts since its more eponymous.